amplify/backend/function/bridgeXYZMutation/.env.example

```diff
@@ -0,0 +1,2 @@
+BRIDGE_API_URL=https://api.sandbox.bridge.xyz
+BRIDGE_API_KEY=
```

Comment:
`.env` files inside lambdas indeed work out of the box, thanks for the heads up @davecreaser!